README.md

```diff
@@ -48,7 +48,7 @@ Installation
 
   * Establish an SSH tunnel to the replicas (only necessary on local environments)
 ```bash
-$ ./php bin/console toolforge:ssh
+$ php bin/console toolforge:ssh
```

Comment:
This should probably match the below and be just `./bin/console …`

Reply:
Done. Thank you!